packages/radix-vue/package.json

```diff
@@ -56,15 +56,31 @@
     "test": "vitest",
     "pub:release": "pnpm publish --access public"
   },
+  "peerDependencies": {
+    "@internationalized/date": "^3.5.2",
```

Comment:
since in the next release we rely on `flat-internationalized-date` isn't better to change it to `flat-internationalized-date`?

Reply:
I dont think that this is correct thing to do in this PR. You can do it in https://github.com/radix-vue/radix-vue/pull/761

Reply:
Yup it's better to handle that in relevant MR 😁